cmd/plume/release.go

```diff
@@ -275,16 +282,36 @@ func doGCE(ctx context.Context, client *http.Client, src *storage.Bucket, spec *
 			gceWaitForImage(pending)
```

Comment:
If I understand this correctly, it means that we would only upload the image once, but on a rerun the old image stays. For AWS we delete the old one and upload the new image.

Reply:
Image is deleted _after_, I guess with such a behavior + image overwriting, we should be safe to rerun.

Reply:
In `len(conflicting) == 1` we can call `api.DeleteImage(image.Name)` and remove the rest of the if branch

Reply:
and then we execute the below unconditionally, not in an `else`

Reply:
This is about the image, at the moment we delete the object at the end while the image stays.

Reply:
Solved by overwriting